api/pkg/apps/webapp/templates/casetype.gohtml

```diff
@@ -57,6 +57,10 @@
                         </select>
                         <label for="team">Select Team</label>
                     </div>
+                    {{if .CaseType}}
```

Comment:
whats that for?

Reply:
values from the disabled selects above we're not getting through

Reply:
maybe it was something to do with the values not being passed along properly, I didn't take the time to investigate to be honest.

looking into it now